build/persistence/postgres/pre-integration-test.sh

```diff
@@ -50,11 +50,12 @@ copy_server_config(){
     cp -pr ${WORKSPACE}/fhir-server/liberty-config-tenants/config/* $DIST/config
     cp -pr ${WORKSPACE}/fhir-server/liberty-config/config/default/fhir-server-config-postgresql.json $DIST/config/default/fhir-server-config.json
 
-    echo "Replacing datasource content in server configDropins..."
-    rm -f $DIST/overrides/datasource-*.xml 2> /dev/null
+    # Note the overrides folder is specifically mounted to the docker image under configDropins/overrides
+    echo "Creating an overrides folder in $DIST"
     mkdir -p $DIST/overrides
-    cp -p ${WORKSPACE}/fhir-server/liberty-config/configDropins/overrides/datasource-bootstrap.xml $DIST/overrides
-    cp -p ${WORKSPACE}/fhir-server/liberty-config/configDropins/overrides/datasource-postgresql.xml $DIST/overrides
+    
+    # Copy and rename the postgres override
+    cp -p ${WORKSPACE}/fhir-server/liberty-config/configDropins/disabled/datasource-postgresql.xml $DIST/overrides/datasource.xml
```

Comment:
I think this is what failed...

Reply:
I think we want the derby (bootstrap) config too because some of the server tests run against different tenants and, for reduced build times, we only provision a postgresql schema for the default tenant.

Reply:
never mind, I see the bootstrap datasources are included in the configDropins/defaults

Reply:
I think I found the issue: a bad change to the datasource definitions.  I think the datasource properties need to be a peer of the jdbcDriver, not a child.  Also fixed indentations so we hopefully don't make that mistake again.